webapp/src/components/Metrics/PerformanceAnalysisTable.tsx

```diff
@@ -0,0 +1,658 @@
+        {
+          ...(props.row.id !== OVERALL_ROW_ID && {
+            [selectedMetricPerFilterOption]: [
+              props.row.basePipeline.filterValue,
+            ],
+          }),
+          ...pipeline,
+        }
+      )}`}
+    >
+      <GridRow {...props} />
+    </Link>
+  );
+
+  const LabelHeader = () => (
+    <Box
+      position="relative"
+      height={LABEL_HEIGHT}
+      width={LABEL_WIDTH}
+      top={LABEL_HEIGHT}
+      padding={(theme) => theme.spacing(1)}
+      zIndex={(theme) => theme.zIndex.mobileStepper}
+      bgcolor={(theme) => theme.palette.background.paper}
+    >
+      <Select
+        sx={{
+          fontFamily: "inherit",
+          fontSize: "inherit",
+          fontWeight: "bold",
+          color: "inherit",
+        }}
+        // We must stop the blur event because if not it causes exceptions elsewhere
+        // See here: https://github.com/mui/mui-x/issues/1439
+        // onBlur={(event) => event.stopPropagation()}
+        variant="standard"
+        id="filter-by-select"
+        value={selectedMetricPerFilterOption}
+        onChange={(event) =>
+          setSelectedMetricPerFilterOption(
+            event.target.value as FilterByViewOption
+          )
+        }
+      >
+        {BASIC_FILTER_OPTIONS.map((key) => (
+          <MenuItem key={key} value={key}>
+            {OPTION_PRETTY_NAME[key]}
+          </MenuItem>
+        ))}
+        <ListSubheader>Smart Tags</ListSubheader>
+        {SMART_TAG_FAMILIES.map((key) => (
+          <MenuItem key={key} value={key}>
+            <Box display="flex" gap={1}>
+              {OPTION_PRETTY_NAME[key]}
+              {React.createElement(SMART_TAG_FAMILY_ICONS[key])}
+            </Box>
+          </MenuItem>
+        ))}
+      </Select>
+    </Box>
+  );
+
+  return errorFetchingBasePipelineData ? (
+    <Typography
+      sx={{ minHeight: 20 }}
+      variant="body2"
+      margin={2}
+      alignSelf="center"
+    >
+      {errorFetchingBasePipelineData.message}
+    </Typography>
+  ) : errorFetchingComparedPipelineData ? (
+    <Typography
+      sx={{ minHeight: 20 }}
+      variant="body2"
+      margin={2}
+      alignSelf="center"
+    >
+      {errorFetchingComparedPipelineData.message}
+    </Typography>
+  ) : (
+    <Box
+      border={(theme) => `1px solid ${theme.palette.grey[200]}`}
+      height="100%"
+      padding={4}
+      marginTop={2}
+    >
+      <Box marginBottom={1} display="flex" justifyContent="space-between">
+        <Box width={340}>
+          <DatasetSplitToggler
+            availableDatasetSplits={availableDatasetSplits}
+            value={selectedDatasetSplit}
+            onChange={setSelectedDatasetSplit}
+          />
+        </Box>
+        {config && (
+          <Box display="flex" flexDirection="row" alignItems="center">
+            <FormControlLabel
+              label={`Compare Baseline (${
+                config?.pipelines?.[pipeline.pipelineIndex].name
+              } ) with:`}
```

Comment:
Let's remove that space:
```suggestion
              }) with:`}
```

Reply:
Ohh!!. Yes!..